src/server/api/workable.js

```diff
@@ -11,13 +11,12 @@ export default class Workable {
         'Content-Type': 'application/json'
       }
     }).then((response) => {
-      const filtered = response.jobs.map((job) => {
+      return response.body.jobs.map((job) => {
```

Comment:
@lithin you make a good point. There's definitely a bit of feature envy going on with this method, i.e. it knows more about the data structure it is passed than it might need to know about. Maybe it could just receive jobs?